swatch-metrics/deploy/clowdapp.yaml

```diff
@@ -391,140 +399,28 @@ objects:
+                    name: swatch-psks
+                    key: self
+              # This env property is necessary to pass the test check.
               - name: SPRING_PROFILES_ACTIVE
-                value: metering-job,kafka-queue
-              - name: JAVA_MAX_MEM_RATIO
-                value: '85'
-              - name: GC_MAX_METASPACE_SIZE
-                value: '256'
-              - name: LOG_FILE
-                value: /logs/server.log
-              - name: LOGGING_LEVEL_ROOT
-                value: ${LOGGING_LEVEL_ROOT}
-              - name: LOGGING_LEVEL_ORG_CANDLEPIN
-                value: ${METRICS_CRON_LOGGING_LEVEL}
-              - name: OPENSHIFT_METERING_RANGE
-                value: ${OPENSHIFT_METERING_RANGE}
-              - name: HOURLY_TALLY_OFFSET
-                value: ${HOURLY_TALLY_OFFSET}
-              - name: DATABASE_HOST
-                valueFrom:
-                  secretKeyRef:
-                    name: swatch-tally-db
-                    key: db.host
-              - name: DATABASE_PORT
-                valueFrom:
-                  secretKeyRef:
-                    name: swatch-tally-db
-                    key: db.port
-              - name: DATABASE_USERNAME
-                valueFrom:
-                  secretKeyRef:
-                    name: swatch-tally-db
-                    key: db.user
-              - name: DATABASE_PASSWORD
-                valueFrom:
-                  secretKeyRef:
-                    name: swatch-tally-db
-                    key: db.password
-              - name: DATABASE_DATABASE
-                valueFrom:
-                  secretKeyRef:
-                    name: swatch-tally-db
-                    key: db.name
-              - name: INVENTORY_DATABASE_HOST
-                valueFrom:
-                  secretKeyRef:
-                    name: ${INVENTORY_SECRET_KEY_NAME}
-                    key: db.host
-              - name: INVENTORY_DATABASE_DATABASE
-                valueFrom:
-                  secretKeyRef:
-                    name: ${INVENTORY_SECRET_KEY_NAME}
-                    key: ${INVENTORY_SECRET_KEY_NAME_PREFIX}name
-              - name: INVENTORY_DATABASE_USERNAME
-                valueFrom:
-                  secretKeyRef:
-                    name: ${INVENTORY_SECRET_KEY_NAME}
-                    key: db.user
-              - name: INVENTORY_DATABASE_PASSWORD
-                valueFrom:
-                  secretKeyRef:
-                    name: ${INVENTORY_SECRET_KEY_NAME}
-                    key: ${INVENTORY_SECRET_KEY_NAME_PREFIX}password
-              - name: PROM_URL
-                value: ${PROM_URL}
-              - name: OTEL_EXPORTER_OTLP_ENDPOINT
-                value: ${OTEL_EXPORTER_OTLP_ENDPOINT}
-              - name: OTEL_SERVICE_NAME
-                value: ${OTEL_SERVICE_NAME}
-              - name: OTEL_JAVAAGENT_ENABLED
-                value: ${OTEL_JAVAAGENT_ENABLED}
+                value: metering-job
```

Comment:
Could you explain why "this env property is necessary to pass the test check".  It's just a little confusing to delete the `metering-job` and `metrics-rhel-job` profiles, but still have a need to set them.

Reply:
Updated

Reply:
Without this property, the PR check fails because it explicitly searches for this property and this value. I've just removed it, so you can see it.